src/utils/discordResponse.ts

```diff
@@ -0,0 +1,10 @@
+import JSONResponse from "./JsonResponse";
+
+export const DISCORD_RESPONSE = (reply: string): JSONResponse => {
+  return new JSONResponse({
+    type: 4,
```

Comment:
what does this type signify

Reply:
Type 4 specifies text reply

Reply:
where can I find all the other types and what they mean.

Reply:
https://discord.com/developers/docs/interactions/receiving-and-responding#interaction-response-object-interaction-callback-type

You can read more about callback type here